src/test/java/uk/ac/ebi/subs/ena/submission/FullSubmissionServiceTest.java

```diff
@@ -44,7 +47,7 @@
     @Value("${ena.password}")
     String ftpPassword;
 
-    public static final int SUBMITTABLE_COUNT = 10;
+    public static final int SUBMITTABLE_COUNT = 1;
```

Comment:
Why did you change it to 1?

Reply:
thanks for catching this. i temporarily changed to generate some xmls as an example for ENA. i ll set it back to 10.